test/repl.jl

```diff
@@ -177,29 +203,6 @@ begin
     hp = REPL.REPLHistoryProvider(Dict{Symbol,Any}(:julia => repl_mode,
                                                    :shell => shell_mode,
                                                    :help  => help_mode))
-    fakehistory =
-    """
-    # time: 2014-06-30 17:32:49 EDT
-    # mode: julia
-    \tshell
-    # time: 2014-06-30 17:32:59 EDT
-    # mode: shell
-    \tll
-    # time: 2014-06-30 17:32:49 EDT
-    # mode: julia
-    \t1 + 1
-    # time: 2014-06-30 17:35:39 EDT
-    # mode: foobar
-    \tbarfoo
-    # time: 2014-06-30 18:44:29 EDT
-    # mode: shell
-    \tls
-    # time: 2014-06-30 19:44:29 EDT
-    # mode: foobar
-    \tls
-    # time: 2014-06-30 20:44:29 EDT
-    # mode: julia
-    \t2 + 2"""
```

Comment:
I really don't think the unindent code should be changing the indentation of these lines at all. The behavior of this was intended to be that if and only if the last line in the double quotes are pure white space, is that whitespace stripped from each preceding line of the quoted data. Perhaps that's not what got implemented though.